src/flashloan/FlashLoanProvider.sol

```diff
@@ -31,6 +33,8 @@ library FlashLoanProvider {
             BalancerFlashLoan.takeFlashLoan(tokens, amounts);
         } else if (flp == FlashLoanProviders.UNISWAPV2) {
             UniswapV2FlashLoan.takeFlashLoan(tokens, amounts);
+        } else if (flp == FlashLoanProviders.AAVEV2) {
+            AAVEV3FlashLoan.takeFlashLoan(tokens, amounts);
```

Comment:
Looks like this should be calling AAVEV2FlashLoan lib

Reply:
Oops, updated.